strongbox-web-core/META-INF/spring/logging-core-context.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where did these two come from? Is this a fix for that excessive logging banner, or what? :)